src/client/api/login.js

```diff
@@ -17,7 +17,12 @@ function login (userData) {
       localStorage.setItem('userId', response.data.userId);
     })
     .catch(function (error) {
-      alert(error.message);
+      toast(`Something went wrong! Oops`, {
```

Comment:
I think usually its the opposite order: `Oops, something went wrong!`

Reply:
Also you should put all the messages in a constants file so that it's easier to change and it'll be standardised.

Reply:
Alright, will change message to `Oops, something went wrong!`. 

I used find and replace to do the toasts as I didn't have time to abstract it out - there were a lot of alerts in the codebase

Reply:
done. Did not move to constants tho